admin-manual/installation/installation.rst

```diff
@@ -294,9 +294,82 @@ If you have trouble with the gearman command try this as an alternative:
 Install from source
 -------------------
 
-Installing from source has been tested using ansible scripts. Ansible installations have been tested for new installations but are not fully tested for upgrades.
+Installing from source has been tested using ansible scripts. Ansible 
+installations have been tested for new installations but are not fully tested 
+for upgrades.
+
+While installing using Vagrant is well suited to trying out Archivematica, it is 
+not suitable for production deployment.
```

Comment:
suggest changing:
While installing using Vagrant is well suited to trying out Archivematica, it is not suitable for production deployment

too:
These instructions are designed to create a test environment on your local machine.  The ansible roles referenced here can be used in production deployments by creating your own ansible playbook to run them.  See https://github.com/artefactual/deploy-pub/playbooks/archivematica for more details.